src/main/java/net/minecraftforge/registries/DeferredRegister.java

```diff
@@ -56,15 +59,35 @@
  */
 public class DeferredRegister<T extends IForgeRegistryEntry<T>>
 {
-    private final IForgeRegistry<T> type;
+    public static <B extends IForgeRegistryEntry<B>> DeferredRegister<B> create(IForgeRegistry<B> reg, String modid)
+    {
+        return new DeferredRegister<B>(reg, modid);
+    }
+
+    public static <B extends IForgeRegistryEntry<B>> DeferredRegister<B> create(Class<B> base, String modid)
```

Comment:
```suggestion
    /**
     * Use for custom registries. Provides helpers to make the registry.
     *
     * @see #makeRegistry(String, Supplier)
     * @see #register(IEventBus)
     */
    public static <B extends IForgeRegistryEntry<B>> DeferredRegister<B> create(Class<B> base, String modid)
```

Reply:
In java it isn't `@see THING` it is `{@link THING}`

Reply:
Hmm it worked though. Gave me a link to the function.

Reply:
Ah turns out java also has a `@see` I just have never actually seen it used before so didn't realize it was actually a thing.